goc/usr/lib/systemd/system/oasis-replica-initclean.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Oasis replica initialization boot cleanup
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
```

Comment:
Why `RemainAfterExit`?

Reply:
That means the service is considered active even after it finishes running.  It doesn't mean much in this case.  It was just in the config that I started from.